### ISSUE TITLE
Question about paper 

### ISSUE THREAD

**Issue by diaodeyi** [microsoft/NUWA#8]
As the paper says in appendix：
 "For example, for long videos or high-resolution frames with large h, w, s, usually (e^h)*(e^w)*(e^s)< (h + w + s)"
Is there any situation that  (e^h)*(e^w)*(e*s)< (h + w + s)?

**Comment by diaodeyi** [comment]
By the way, i want to konw  why does the equation 7 in paper use e^x not just use x for the difinition?
in this way the complexity of 3DNA is (hws)(h+w+s)
<img width="534" alt="截屏2022-04-13 上午10 43 01" src="https://user-images.githubusercontent.com/48873381/163089661-216eba21-8351-47ce-a03b-1ed90b470efd.png">

**Comment by diaodeyi** [comment]
这里的HWS是范围吧，那e的HWS次方不是早就超出attention的范围了？

**Comment by woctezuma** [comment]
> Is there any situation that (e^h)_(e^w)_(e^s)< (h + w + s)?

![Picture](https://i.imgur.com/lezGaTe.png)

Reference: [Wolfram Alpha](https://www.wolframalpha.com/input?i=find+h%2Cw%2Cs+such+that+%28e%5Eh%29%28e%5Ew%29%28e%5Es%29%3C+%28h+%2B+w+%2B+s%29).

**Comment by woctezuma** [comment]
![Table](https://i.imgur.com/TxZxX4k.png)

![Picture](https://i.imgur.com/Z0Ymr33.png)

Indeed, this is a weird statement. By rewriting the product of exponential factors as a single exponential of the variable `x=h+w+s`:

```
(e^h)(e^w)(e^s) = e^(h+w+s)
```

We are on a classical inequality between `x` and `e^x`, and it is easy to prove or verify that `x < e^x` for every `x`.

**Comment by diaodeyi** [comment]
> 

yeah,so how it would be  x > e^x.  (h+w+s)> e^h* e^w *e^s. like the paper said.
